encryption-service-vault/src/main/java/com/mx/path/facility/security/vault/VaultEncryptionServiceConfiguration.java

```diff
@@ -0,0 +1,81 @@
+package com.mx.path.facility.security.vault;
+
+import java.time.Duration;
+
+import lombok.Data;
+
+import com.mx.common.configuration.ConfigurationField;
+
+@Data
+public class VaultEncryptionServiceConfiguration {
+
+  private static final int DEFAULT_ENGINE_VERSION = 1;
+  private static final String DEFAULT_URI = "http://127.0.0.1:8200";
+  private static final int DEFAULT_MAX_RETRIES = 0;
+  private static final int DEFAULT_RETRY_INTERVAL_MILLISECONDS = 200;
+  private static final String DEFAULT_AUTHENTICATION = "APPID";
+  private static final String DEFAULT_KEY_NAME = "vault_session";
+  private static final int DEFAULT_NUM_KEYS_TO_KEEP_COUNT = 1;
+
+  @ConfigurationField
+  private boolean enabled = true;
+
+  @ConfigurationField
+  private int engineVersion = DEFAULT_ENGINE_VERSION;
+
+  @ConfigurationField
+  private String keyName = DEFAULT_KEY_NAME;
+
+  @ConfigurationField
+  private int maxRetries = DEFAULT_MAX_RETRIES;
+
+  @ConfigurationField
+  private int numKeysToKeep = DEFAULT_NUM_KEYS_TO_KEEP_COUNT;
+
+  /**
+   * @#deprecated use retryIntervalMilliseconds
```

Comment:
Isn't this comment saying to 'use' the same field as this one?

Reply:
Ooops. That was a mistake. Fixing.